pkg/endpoint/local_endpoint_test.go

```diff
@@ -120,6 +120,12 @@ var _ = Describe("GetLocalSpec", func() {
 			Gw:        net.ParseIP("2001:0:0:0::"),
 		})).To(Succeed())
 
+		Expect(netLink.RouteAdd(&netlink.Route{
```

Comment:
To simulate ovnk use case, I think that you should add the route before 

https://github.com/submariner-io/submariner/blob/0f0696db23adf1666100dc3050d89f4af16fc703/pkg/endpoint/local_endpoint_test.go#L117